zcash/src/private_key.rs

```diff
@@ -60,7 +63,18 @@ impl<N: ZcashNetwork> Display for P2PKHSpendingKey<N> {
 pub struct P2SHSpendingKey {}
 
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
-pub struct SproutSpendingKey {}
+pub struct SproutSpendingKey {
+    pub(crate) spending_key: [u8; 32],
```

Comment:
If its just one variable of same terminology, I suggest `pub struct SproutSpendingKey([u8; 32]) for cleaner convention